app/[slug]/settings/users/page.tsx

```diff
@@ -2,15 +2,15 @@ import { redirect } from "next/navigation";
 
 import SettingsNav from "@/app/(main)/settings/settings-nav";
 import UserSettings from "@/app/(main)/settings/users/user-settings";
-import { getMembersByTenantId } from "@/lib/server/service";
+import { getAuthenticatedMembersByTenantId } from "@/lib/server/service";
 import { adminOrRedirect } from "@/lib/server/utils";
 interface Props {
   params: { slug: string };
 }
 
 export default async function UsersSettingsPage({ params }: Props) {
   const { tenant } = await adminOrRedirect();
-  const members = await getMembersByTenantId(tenant.id);
+  const members = await getAuthenticatedMembersByTenantId(tenant.id);
```

Comment:
Kind of wordy but I see what you are going for.  Members is already an ephemeral concept so I think it's ok to just keep the name shorter and change the implementation.